src/components/WheelPicker/index.tsx

```diff
@@ -222,7 +222,7 @@ const WheelPicker = ({
   const fakeLabelProps = useMemo(() => {
     return {...labelMargins, ...labelProps};
   }, [labelMargins, labelProps]);
-
+  const itemLabelStyle = useMemo(() => ({...labelStyle, opacity: 0}), [labelStyle]);
```

Comment:
I'd put it in the `item` itself.
I'd probably write it as `[labelStyle, {opacity}]` but now I've seen [this article](https://medium.com/@fjmorant/styles-and-performance-in-react-native-unlocking-the-best-of-both-worlds-2263632a12bb):
> 4. Best Practices for Styling Performance
...
Flattening Styles: Using arrays for styles can be computationally expensive. Instead, use StyleSheet.flatten to merge style objects.

So maybe more investigation is needed.